src/App.js

```diff
@@ -1,7 +1,72 @@
-import React from "react";
+import React, { useEffect, useState } from 'react';
+import jwt_decode from 'jwt-decode';
+import Parse from 'parse';
 
-const App = () => {
-  return <h1>Hello React!</h1>;
-};
+function App() {
```

Comment:
App is reserved as an entry point. 
Create another component for log in.
Also, use arrow functions not pure functions whenever possible as its standard practice for es6